### DISCUSSION TITLE
n00b question: trying to get off the ground with this wonderful new purplish project

### DISCUSSION THREAD

**Discussion by antonymott** [CanadaHonk/porffor#213] [General]
Hi,

first: thanks to CanadaHonk and other contributors so much for going after this very purplish goal...exciting!

If you could please point me, without doubt the n00biest porffor n00b in this discussion, in the right direction, I'm stumped trying to test locally after playing in the playground https://porffor.dev

1. After installing the package in my project folder (pnpm add -D @porffor/latest) I wrote the simplest porffor_test.ts file, a naive guess, and it looks like this:
```typescript
import { Porffor } from 'porffor'

let arr = [ 2, 4, 8, 16 ];

for (let i = 0; i < arr.length; i++) Porffor.numberLog(arr[i]);
```
Porrfor looked at first like a type, but I see it's used as a Class simply to log the output to console (I guess). Yet there appears no export named "Porffor" so vscode's squiggly red line doesn't let me get to the exciting first stage of typing:
```sh
porff
```

2. my 1st goal is to just tinker around, add additional complexity and functionality to a .ts (or perhaps it has to be .js?) file, run in the terminal, and also compile to wasm target, until something breaks.
3. Being purplish, I translated "...does not use an import standard like WASI, mostly unusable on its own" to mean => "wasm target is somewhat useable"! So what hack(s) do I need to add, as a specific example, to get the simple array example WASM target to glue into an html file so I can serve on localhost?
5. from other contributors POV, my more lofty goal is to do eventually contribute, at first by sharing stuff that hasn't already been tried by the contributors, in case it helps either your team, or other first-timers to porffor not to run away.

Oh, and yes, I looked with great excitement at the demos, particularly Mandelbrot.bf [https://github.com/CanadaHonk/porffor/blob/main/demos/bf/mandelbrot.bf](Mandelbrot), but wanted to start with the most basic example I'd seen on the playground.

Thanks again...for embarking on one of the most startlingly unusual projects I've seen, not just because of the goal, but it's really a David and Goliath story.

Inspiring. Inspired!

Best, Antony

**Comment by CanadaHonk** [comment]
Hi, thanks for trying it out! You can just use the standard `console.log` (I should update the playground). Porffor just exists as a global (like process for Node), you don't have to import it. You can just do `porf foo.ts` (or .js) to run a file, like Node.

**Reply by antonymott** [reply]
Yep, that did it. Thanks.
